samples/java-spring-eventsourced-counter/README.md

```diff
@@ -1,52 +1,53 @@
-# java-spring-eventsourced-counter
-
+# Event Sourecd Counter
 
+## Designing
 
 To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/java/development-process.html) in the documentation.
 
+## Developing
 
+This project demonstrates the use of Value Entity and View components.
+To understand more about these components, see [Developing services](https://docs.kalix.io/services/)
+and in particular the [Java section](https://docs.kalix.io/java/)
 
-This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation. Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
-
-
+## Building
 
 Use Maven to build your project:
 
 ```shell
 mvn compile
 ```
 
+## Running Locally
 
+When running a Kalix application locally, at least two applications are required. The current Kalix application and its companion Kalix Proxy.
 
-To run the example locally, you must run the Kalix proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
```

Comment:
How about a section `## Debugging` that will inform users to run docker-compose and the app manually?

Reply:
I would do it later. We need to have a better story for this. 
I have something in my mind and I will create a ticket to discuss it further. Stay tuned...

Reply:
See https://github.com/lightbend/kalix-jvm-sdk/issues/1631